bc_obps/registration/schema/operator.py

```diff
@@ -52,8 +52,6 @@ class OperatorIn(ModelSchema):
     Schema for the Operator model
     """
 
-    user_operator_id: UUID
```

Comment:
This isn't needed anymore since we are declining all User Operators tied to that Operator.